dll/shellext/fontext/CFontExt.cpp

```diff
@@ -363,3 +365,176 @@ STDMETHODIMP CFontExt::GetClassID(CLSID *lpClassId)
     return S_OK;
 }
 
+// *** IDropTarget methods ***
+STDMETHODIMP CFontExt::DragEnter(IDataObject* pDataObj, DWORD grfKeyState, POINTL pt, DWORD* pdwEffect)
+{
+    FORMATETC formatETC;
+    HRESULT hr = DoGetFormatETC(pDataObj, formatETC);
+    if (hr != S_OK)
+    {
+        *pdwEffect = DROPEFFECT_NONE;
```

Comment:
is `pdwEffect` always a required parameter?

Reply:
https://docs.microsoft.com/en-us/windows/win32/api/oleidl/nf-oleidl-idroptarget-dragenter
>On return, the method must write the effect, one of the DROPEFFECT flags, to the pdwEffect parameter.